molecule/default/prepare.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Converge
+  hosts: all
+  tasks:
+    - name: Download OpenShift Binary
+      unarchive:
+        src: "{{ oc_binary_url }}"
+        dest: "/usr/local/bin"
+        remote_src: yes
+    - name: Manage OpenShift Configuration File
+      block:
+        - name: Copy OpenShift Configuration File
+          copy:
+            src: "~/.kube/config"
+            dest: "~/.kube/"
+      when: use_kube_file|bool
+    - name: Manage OpenShift Configuration File via Login
```

Comment:
Any chance we can leverage the [openshift-login role](https://github.com/redhat-cop/casl-ansible/tree/master/roles/openshift-login)  ?

Reply:
With regard to the indenting, yes we are violating the indentation rule as the rules that are being used by yamllint extend the default set of rules. If we are to expand molecule and or linting across the set of repositories, additional configurations would need to be made to go against most yaml policies. I would recommend that we revisit the topic.